os/binfmt/binfmt_execmodule.c

```diff
@@ -205,7 +205,7 @@ int exec_module(FAR struct binary_s *binp)
 
 	tcb = (FAR struct task_tcb_s *)kmm_zalloc(sizeof(struct task_tcb_s));
 	if (!tcb) {
-		return -ENOMEM;
+		goto errout_with_appheap;
```

Comment:
In `errout_with_appheap`, it returns `ret` but there is no assigning of `ret`.
How about initializing or assigning the `ret` from the first or here?

Reply:
Done.